### ISSUE TITLE
Create EventImportWorker

### ISSUE THREAD

**Issue by wendelfabianchinsamy** [datacite/datacite#2241]
Create an EventImportWorker in Levriero. This worker will consume messages from the events queue and fire off a POST /events request.

Why do this in Levriero?
 
I did consider doing this as a separate worker process (perhaps a AWS Lambda), however, for the most part, the Levriero containers are put under very little strain and I think moving to an AWS Lambda would be wasteful. Let's use existing unused resources and we can re-think this approach in the future, if perhaps we move some of these types processing into external processes.

**Comment by digitaldogsbody** [comment]
I think that moving away from the creation being handled by Lupo should be the longer-term target (i.e no more POSTs).

It seems like overkill to add the model+controller functionality (and all the associated cruft like the DB connection, queueing the index jobs etc) to Levriero if we are still thinking to completely separate DOI registration and everything else, so your approach here feels like a good compromise.

Maybe as a secondary step we could redistribute the load and run a dedicated Lupo container for handling Event creation, in the same way that we now have the queue-worker containers (maybe we could even use them??) - i.e it would be POST to `http://event-creator.local/events` rather than `http://client-api.local/events`, what do you think?

**Comment by wendelfabianchinsamy** [comment]
> I think that moving away from the creation being handled by Lupo should be the longer-term target (i.e no more POSTs).
> 
> It seems like overkill to add the model+controller functionality (and all the associated cruft like the DB connection, queueing the index jobs etc) to Levriero if we are still thinking to completely separate DOI registration and everything else, so your approach here feels like a good compromise.
> 
> Maybe as a secondary step we could redistribute the load and run a dedicated Lupo container for handling Event creation, in the same way that we now have the queue-worker containers (maybe we could even use them??) - i.e it would be POST to `http://event-creator.local/events` rather than `http://client-api.local/events`, what do you think?

Yes I certainly think that could be the next step i.e. having a dedicated container for handling these POST requests. This approach will also help us realise the template for breaking up Lupo. Since handling event creation and indexing does not require all of Lupo (in it's current form) but only a subset of it's functionality.
